PlaybackLab/FullscreenSampleApp/app/src/main/java/com/ooyala/sample/MultiplePlayerActivity.java

```diff
@@ -0,0 +1,208 @@
+            return centerView != null ? layoutManager.getPosition(centerView) : RecyclerView.NO_POSITION;
+        }
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_players);
+        ButterKnife.bind(this);
+
+        dataList = Constants.populateData();
+
+        scrollListener = new ScrollListener();
+        scrollListener.setSnapPosition(0);
+
+        playerAdapter = new MultiplePlayerAdapter(dataList, this, recyclerView);
+        playerAdapter.setAutoPlayIndex(0);
+
+        layoutManager = new LinearLayoutManager(this, LinearLayoutManager.HORIZONTAL, false);
+        recyclerView.setLayoutManager(layoutManager);
+        recyclerView.addOnScrollListener(scrollListener);
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setDrawingCacheEnabled(true);
+        recyclerView.setDrawingCacheQuality(View.DRAWING_CACHE_QUALITY_LOW);
+        recyclerView.setAdapter(playerAdapter);
+
+        snapHelper = new PagerSnapHelper();
+        snapHelper.attachToRecyclerView(recyclerView);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+        if (scrollListener != null) {
+            scrollListener.destroy();
+        }
+
+        if (playerAdapter != null) {
+            updatePlayerList(MediaPlayer::onDestroy);
+            playerAdapter.destroy();
+        }
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onStart() {
+        super.onStart();
+        updatePlayerList(MediaPlayer::onStart);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onStop() {
+        super.onStop();
+        updatePlayerList(MediaPlayer::onStop);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onPause() {
+        super.onPause();
+        updatePlayerList(MediaPlayer::onPause);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    protected void onResume() {
+        super.onResume();
+        updatePlayerList(MediaPlayer::onResume);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        updatePlayerList(MediaPlayer::onBackPressed);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
+
+        boolean isFullScreen = newConfig.orientation == SCREEN_ORIENTATION_USER;
+        textView.setVisibility(isFullScreen ? View.GONE : View.VISIBLE);
+        playerAdapter.setFullscreenMode(isFullScreen);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    private void updatePlayerList(Consumer<MediaPlayer> method) {
```

Comment:
I suggest to use import androidx.core.util.Consumer; 
it will work with lower version of api